.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
     "editor.formatOnSave": true,
+    "ansible.python.interpreterPath": "/home/jaydee/git/kubeseal-webgui/api/.venv/bin/python",
```

Comment:
Probably not correct for everyone. Might use a relative path or a reference to the workspace (e.g. `${workspaceFolder}`)?

Reply:
Added it to gitignore.